pkg/collector/corechecks/snmp/internal/devicecheck/devicecheck_test.go

```diff
@@ -561,11 +563,21 @@ profiles:
 	deviceCk, err := NewDeviceCheck(config, "1.2.3.4", sessionFactory)
 	assert.Nil(t, err)
 
+	snmpTags := []string{"snmp_device:1.2.3.4", "snmp_profile:f5-big-ip", "device_vendor:f5", "snmp_host:foo_sys_name",
+		"static_tag:from_profile_root", "static_tag:from_profile_root", "some_tag:some_tag_value", "prefix:f", "suffix:oo_sys_name"}
```

Comment:
is the duplicated `static_tag:from_profile_root` tag expected?

Reply:
it was an error, but I changed one of them to `static_tag:from_base_profile` to add this test